packages/headless/src/controllers/core/generated-answer/headless-searchapi-generated-answer.test.ts

```diff
@@ -124,26 +124,34 @@ describe('searchapi-generated-answer', () => {
     expect(closeGeneratedAnswerFeedbackModal).toHaveBeenCalled();
   });
 
-  it('dispatches a send feedback action', () => {
+  // it('dispatches a send feedback action', () => {
+  //   const generatedAnswer = createGeneratedAnswer();
+  //   const feedback: GeneratedAnswerFeedback = 'harmful';
+  //   generatedAnswer.sendFeedback(feedback);
+  //   expect(
+  //     generatedAnswerAnalyticsClient.logGeneratedAnswerFeedback
+  //   ).toHaveBeenCalledWith(feedback);
+  //   expect(sendGeneratedAnswerFeedback).toHaveBeenCalledTimes(1);
+  // });
```

Comment:
TODO?

Reply:
this is no longer needed, forgot to remove 👍